packages/ethereum-wallets/src/lib/index.ts

```
@@ -861,12 +899,28 @@ const EthereumWallets: WalletBehaviourFactory<
           logger.log("Wallet already connected");
         }
 
-        if (isBannedNearAddress(address.toLowerCase())) {
+        console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)